eng/tox/run_coverage.py

```diff
@@ -59,6 +56,21 @@ def get_total_coverage(coverage_file: str) -> Optional[float]:
                 # before the 'logmetric' start string.
                 print(f'logmetric: {json.dumps(metric_obj)}')
 
-    if is_check_enabled(args.target_package, "cov_enforcement", False):
-        logging.info("Coverage enforcement is enabled for this package.")
+            # todo: add relative coverage comparison after we generate eng/coverages.json file in main
+            # when we add that, we will call the config setting relative_cov to check if this is enabled
+            # there, we will only ever compare the baseline % against the generated %
+
+            # as a fallback, we can always check the absolute coverage % against the config setting
+            # if the config setting is not set, we will not enforce any coverage
+            if is_check_enabled(args.target_package, "absolute_cov", False):
+                logging.info("Coverage enforcement is enabled for this package.")
+
+                # if this threshold is not set in config setting, the default will be very high
+                cov_threshold = get_config_setting(args.target_package, "absolute_cov_percent", 95.0)
+                if total_coverage < float(cov_threshold):
+                    logging.error(
+                        f"Coverage for {pkg_details.name} is below the threshold of {cov_threshold:.2f}% (actual: {total_coverage:.2f}%)"
+                    )
+                    # exit(1)
```

Comment:
Should we restore this (assuming it was commented for testing)?
```suggestion
                    exit(1)
```

Reply:
100%. Doing a cleanup pass to get this PR merged.